### PR TITLE
docs: Update installation instructions for MSYS2

### LINKED ISSUE #13229: `error: target not found: mingw-w64-ucrt-x86_64-zed` on windows11 pacman installation through UCRT64
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Hi, 
I've been following the installation instructions from the doc file.
I've installed MSYS2 from the official site and executed the following command:
```bash
nsant@DESKTOP-5M0DTDC UCRT64 ~
$ pacman -S mingw-w64-ucrt-x86_64-zed
```
but I've been getting the following error:
`error: target not found: mingw-w64-ucrt-x86_64-zed`

### Environment

OS Name	Microsoft Windows 11 Pro
Version	10.0.22635 Build 22635
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	DESKTOP-5M0DTDC
System Manufacturer	SchenkerTechnologiesGmbH
System Model	XMG FUSION 15 (XFU15L19)
System Type	x64-based PC
System SKU	XFU15L19
Processor	Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz, 2592 Mhz, 6 Core(s), 12 Logical Processor(s)
BIOS Version/Date	Intel Corp. QCCFL357.0144.2022.0124.1433, 24/01/2022
SMBIOS Version	3.2
Embedded Controller Version	1.33
BIOS Mode	UEFI
BaseBoard Manufacturer	Intel Corporation
BaseBoard Product	LAPQC71A
BaseBoard Version	K54899-305
Platform Role	Mobile
Secure Boot State	Off
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\WINDOWS
System Directory	C:\WINDOWS\system32
Boot Device	\Device\HarddiskVolume1
Locale	United States
Hardware Abstraction Layer	Version = "10.0.22621.2706"
User Name	DESKTOP-5M0DTDC\nsant
Time Zone	W. Europe Daylight Time
Installed Physical Memory (RAM)	32,0 GB
Total Physical Memory	31,9 GB
Available Physical Memory	16,4 GB
Total Virtual Memory	36,6 GB
Available Virtual Memory	15,2 GB
Page File Space	4,75 GB

### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

_No response_

### PR DESCRIPTION
**Pull request by fenix-hub**
Fixes #13229 

Release Notes:

- N/A

### CODE REVIEW

**Comment by fenix-hub** [comment]
@cla-bot check

File: docs/src/development/windows.md [hunk 1]
```diff
@@ -69,7 +69,8 @@ cargo test --workspace
 [MSYS2](https://msys2.org/) distribution provides Zed as a package. To download the prebuilt binary, run
 
 ```
-pacman -S mingw-w64-ucrt-x86_64-zed
+$ pacman -Syu
```
Comment (maxdeviant) [inline docs/src/development/windows.md:72]: We don't prefix any of our shell commands with `$` in docs.

```suggestion
pacman -Syu
```

File: docs/src/development/windows.md [hunk 2]
```diff
@@ -69,7 +69,8 @@ cargo test --workspace
 [MSYS2](https://msys2.org/) distribution provides Zed as a package. To download the prebuilt binary, run
 
 ```
-pacman -S mingw-w64-ucrt-x86_64-zed
+pacman -Syu
+$ pacman -S mingw-w64-ucrt-x86_64-zed
```
Comment (maxdeviant) [inline docs/src/development/windows.md:73]:
```suggestion
pacman -S mingw-w64-ucrt-x86_64-zed
```

**Comment by mikayla-maki** [comment]
Thanks!

**Pull request merged**